specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,257 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../document-status.md)
+
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Resource](#resource)
+  * [Resource](#resource-1)
+  * [ResourceVarying](#resourcevarying)
+- [Events](#events)
+  * [PageView](#pageview)
+  * [PageLoad](#pageload)
+  * [PageNavigationTiming](#pagenavigationtiming)
+  * [ResourceTiming](#resourcetiming)
+  * [HTTP](#http)
+    + [HTTP request and response headers](#http-request-and-response-headers)
+  * [HttpRequestTiming](#httprequesttiming)
+  * [Exception](#exception)
+  * [UserAction](#useraction)
+
+<!-- tocstop -->
+
+</details>
+
+
+This document describes the semantic conventions for browser resource and events.
+
+# Resource
+
+## Resource
+<!-- semconv browser.resource -->
+The event name MUST be `BrowserResource`.
```

Comment:
I assume that this is because of the generation from yaml and likewise for `BrowserResourceVarying`, as the current descriptions don't really make sense (because they are not events 😄 )

Reply:
Removing `type: event` got rid of this sentence.